main.py

```diff
@@ -34,8 +34,8 @@ def build_app(g_port):
 
 
 if __name__ == '__main__':
-    clockchain = Clockchain()
     networker = Networker()
+    clockchain = Clockchain(networker)
```

Comment:
Pls keep Clockchain as pure instance without using Networker. Not sure how you can do it but I spent a lot of effort to avoid spaghetti code by making networker and clockchain not call on eachother